Cargo.toml

```diff
@@ -12,7 +12,7 @@ description = "Extism Plug-in Development Kit (PDK) for Rust"
 anyhow = "1"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
-extism-pdk-derive = {path = "./derive"}
+extism-pdk-derive = {version = "0.0.1-rc.1"}
```

Comment:
```suggestion
extism-pdk-derive = {path = "./derive", version = "0.0.1-rc.1"}
```
^ This allows you to use the local version while developing, but uses `version` for releases